src/tests/integration/test-data/foundry/test/FreshBytes.t.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity =0.8.13;
+
+import "forge-std/Test.sol";
+
+import "../src/KEVMCheats.sol";
+
+contract FreshBytesTest is Test, KEVMCheats {
+    bytes1 local_byte;
+    bytes local_bytes;
+
+    function manip_symbolic_bytes(bytes memory b) public {
+        uint middle = b.length / 2;
+        b[middle] = hex'aa';
+    }
+
+    function test_symbolic_bytes_1() public {
+        bytes memory fresh_bytes_1 = kevm.freshBytes(5);
+        local_byte = fresh_bytes_1[0];
+        assertEq(fresh_bytes_1[0], local_byte);
+
+        bytes memory fresh_bytes_2 = kevm.freshBytes(67);
+        assertEq(fresh_bytes_2.length, 67);
+        local_bytes = fresh_bytes_2;
+        assertEq(fresh_bytes_2, local_bytes);
+    }
```

Comment:
Could we have an even more general test, with bounded symbolic lengths and symbolic indexing? The goal is to test all aspects of the "symbolicness" to understand the limits of the prover, together with any lemmas that might need to be introduced.